python/ray/tune/experimental/output.py

```diff
@@ -617,7 +630,8 @@ def setup(
         start_time: Optional[float] = None,
         **kwargs,
     ):
-        self._start_time = start_time
+        if start_time is not None:
```

Comment:
This was a bug. setup gets called with start_time=None, then assigned to `self.start_time`, then crashes due to comparison between None and int (`time.time()`).

Reply:
Oh, did you have a repro for this bug? A user was mentioning it at some point, but I couldn't repro.

Reply:
Sure, it happenes now as soon as you are running with an RLlib Algorithm (and NOT defining your custom progress reporter).

For example:
```
git checkout [this PR]
cd ray/rllib/examples/ray_tune
python custom_progress_reporter.py
```

After a while, when trying to print out a status table (e.g. at the very end of the run).
```
Traceback (most recent call last):
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/execution/tune_controller.py", line 1220, in _on_result
    on_result(trial, *args, **kwargs)
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/execution/tune_controller.py", line 1733, in _on_saving_result
    self._maybe_execute_queued_decision(trial, after_save=True)
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/execution/tune_controller.py", line 1311, in _maybe_execute_queued_decision
    self._execute_action(trial, final_decision, after_save=after_save)
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/execution/tune_controller.py", line 1297, in _execute_action
    self.stop_trial(trial)
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/execution/tune_controller.py", line 1443, in stop_trial
    self._process_trial_failure(
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/execution/tune_controller.py", line 1358, in _process_trial_failure
    self._callbacks.on_trial_error(
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/callback.py", line 422, in on_trial_error
    callback.on_trial_error(**info)
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/experimental/output.py", line 760, in on_trial_error
    curr_time_str, running_time_str = _get_time_str(self._start_time, time.time())
  File "/Users/sven/anaconda3/envs/ray/lib/python3.10/site-packages/ray/tune/experimental/output.py", line 166, in _get_time_str
    start_time_dt = datetime.datetime.fromtimestamp(start_time)
TypeError: 'NoneType' object cannot be interpreted as an integer
```